_inc/client/state/site/reducer.js

```diff
@@ -105,10 +118,26 @@ export function isFetchingSiteData( state ) {
 	);
 }
 
+/**
+ * Returns true if currently requesting site benefits. Otherwise false.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Boolean}       Whether benefits are being requested
+ */
 export function isFetchingSiteBenefits( state ) {
 	return !! state.jetpack.siteData.requests.isFetchingSiteBenefits;
 }
 
+/**
+ * Returns true if currently requesting site benefits. Otherwise false.
```

Comment:
What is the difference between `isFetchingSiteBenefits` and `isFetchingProducts` ?
Do we need boot? This comment should be updated to reflect that we are talking about products.

Reply:
They are for different parts of the state tree, this was just a comment that didn't get changed correctly.